### ISSUE TITLE
Add a license to a repository

### ISSUE THREAD

**Issue by frostasm** [itsnine/yolov5-onnxruntime#15]
First of all, thanks for the working C++ code that uses onnxruntime with yolov5 :+1: 
Could you please add the license file to the repository so that it is clear how your code can be used in other projects

**Comment by zgoey** [comment]
Yes, it would be nice to have a non-restrictive license like MIT (https://choosealicense.com/licenses/mit/).
